### DISCUSSION TITLE
Does `split` copy window only variables to new window ?

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [neovim/neovim#29100] [Q&A]
i have this issue:
- i have window A
- i set `winbar` to window A (`vim.wo[id].winbar = XXX` )
- i `vsplit` and winbar i set for window A is now on window B  AND window A...

Is this intentional ? How can i prevent this ?
`vim.wo[0][0].winbar = "ABC" ` doesn't change the behavior

**Comment by wookayin** [comment]
What do you exactly want? to have the winbar option set for window B and not for window A?

Please try `vim.opt_global` or `vim.go`.
